.github/workflows/build-java.yaml

```diff
@@ -0,0 +1,25 @@
+name: Build Java Protos
+
+on:
+  push:
+    branches:
+      - master
+      - main
+  pull_request:
```

Comment:
Can remove these events if not relevant for this workflow..

Reply:
This is to just verify any build error on proto generating.